include/ftl/fiber.h

```diff
@@ -56,45 +55,41 @@ inline std::size_t RoundUp(std::size_t numToRound, std::size_t multiple);
 
 typedef void (*FiberStartRoutine)(void *arg);
 
-
 class Fiber {
 public:
 	/**
 	 * Default constructor
 	 * Nothing is allocated. This can be used as a thread fiber.
 	 */
-	Fiber()
-		: m_stack(nullptr),
-		  m_systemPageSize(0),
-		  m_stackSize(0),
-		  m_context(nullptr),
-		  m_arg(0) {
+	Fiber() : m_stack(nullptr), m_systemPageSize(0), m_stackSize(0), m_context(nullptr), m_arg(0) {
 	}
 	/**
 	 * Allocates a stack and sets it up to start executing 'startRoutine' when first switched to
 	 *
-	 * @param stackSize        The stack size for the fiber. If guard pages are being used, this will be rounded up to the next multiple of the system page size
+	 * @param stackSize        The stack size for the fiber. If guard pages are being used, this will be rounded up to
+	 * the next multiple of the system page size
 	 * @param startRoutine     The function to run when the fiber first starts
 	 * @param arg              The argument to pass to 'startRoutine'
 	 */
-	Fiber(std::size_t stackSize, FiberStartRoutine startRoutine, void *arg)
-			: m_arg(arg) {
-		#if defined(FTL_FIBER_STACK_GUARD_PAGES)
```

Comment:
If clang format can do it, I would prefer the `#if` statements to use logical indentation as though it was a real `if` statement

Reply:
I don't disagree, clang-format just can't do that at this point.